java/src/apps/AppsBundle.properties

```diff
@@ -3,7 +3,7 @@
 # Default properties for the apps package of jmri.
 
 # general menu items
-#MenuFile = File
+MenuFile = File
```

Comment:
This overrides the MenuFile key in jmri/NamedBeanBundle.properties, and should be commented out or removed.